packages/web-app-files/src/views/Personal.vue

```diff
@@ -1,74 +1,77 @@
 <template>
   <div>
-    <app-bar
-      :has-bulk-actions="true"
-      :breadcrumbs="breadcrumbs"
-      :breadcrumbs-context-actions-items="[currentFolder]"
-      :show-actions-on-selection="true"
-    >
-      <template #actions>
-        <create-and-upload />
-      </template>
-    </app-bar>
-    <app-loading-spinner v-if="loadResourcesTask.isRunning" />
-    <template v-else>
-      <not-found-message v-if="folderNotFound" class="files-not-found oc-height-1-1" />
-      <no-content-message
-        v-else-if="isEmpty"
-        id="files-personal-empty"
-        class="files-empty"
-        icon="folder"
+    <div v-if="isLightweight && isHomeRoute"><home /></div>
```

Comment:
Why render the `Home` view into the `Personal` view and not just make it a dedicated view inkl. route?

Reply:
This is a little bit of a workaround. I don't find a way to remove "Personal" nav item as there should be at least one in export in web/packages/web-app-files/index.js. And this export happens before the user information arrives. Any better ideas on that? I would otherwise love to have an extra route for lightweight personal view